src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,21 @@
 import React from 'react';
+import { User } from '../../types/User';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+import './UserInfo.scss';
+
+interface Props {
+  user: User;
+}
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user;
```

Comment:
Use new lines if you have 3 or more properties
```suggestion
  const { name, email } = user;
```